src/main/scala/com/microsoft/hyperspace/index/IndexCollectionManager.scala

```diff
@@ -63,16 +63,17 @@ class IndexCollectionManager(
     }
   }
 
-  override def refresh(indexName: String): Unit = {
+  override def refresh(indexName: String, mode: String): Unit = {
     withLogManager(indexName) { logManager =>
       val indexPath = PathResolver(spark.sessionState.conf).getIndexPath(indexName)
       val dataManager = indexDataManagerFactory.create(indexPath)
-      if (HyperspaceConf.refreshDeleteEnabled(spark)) {
+      if (mode.equals(REFRESH_MODE_INCREMENTAL)) {
```

Comment:
Can we allow case insensitive comparison?

Reply:
thanks @imback82 , added